cdap-ui/cypress/integration/pluginjsoncreator.spec.ts

```diff
@@ -0,0 +1,651 @@
+              `show-${showIndex}`
+            )} ${dataCy('name')}`
+          )
+            .invoke('val')
+            .then((val) => {
+              expect(val).equals(showVal.name);
+            });
+
+          cy.get(
+            `${filterSelector} ${dataCy('filter-showlist-input')} ${dataCy(
+              `show-${showIndex}`
+            )} ${dataCy('type')}`
+          )
+            .invoke('val')
+            .then((val) => {
+              expect(val).equals(showVal.type);
+            });
+        });
+      }
+    });
+  }
+}
+
+describe('Plugin Information Page', () => {
+  // Uses API call to login instead of logging in manually through UI
+  before(() => {
+    Helpers.loginIfRequired().then(() => {
+      cy.getCookie('CDAP_Auth_Token').then((cookie) => {
+        if (!cookie) {
+          return;
+        }
+        headers = {
+          Authorization: 'Bearer ' + cookie.value,
+        };
+      });
+    });
+    const stub = cy.stub();
+    cy.window().then((win) => {
+      win.onbeforeunload = null;
+    });
+    cy.on('window:confirm', stub);
+
+    Helpers.getArtifactsPoll(headers);
+  });
+
+  describe('Editing data', () => {
+    before(() => {
+      cy.visit('/cdap/ns/default/plugincreation');
+    });
+
+    it('should edit PluginInfoPage', () => {
+      // Fill out plugin name
+      cy.get(dataCy('plugin-name'))
+        .click()
+        .type(MOCK_PLUGIN_INFO.pluginName);
+
+      // Fill out plugin type
+      cy.get(dataCy('select-plugin-type')).click();
+      cy.get(dataCy(`option-${MOCK_PLUGIN_INFO.pluginType}`)).click();
+
+      // Fill out display name
+      cy.get(dataCy('display-name'))
+        .click()
+        .type(MOCK_PLUGIN_INFO.displayName);
+
+      // Validate whether input has been filled
+      cy.get(dataCy('plugin-name'))
+        .invoke('val')
+        .then((val) => {
+          expect(val).equals(MOCK_PLUGIN_INFO.pluginName);
+        });
+      cy.get(dataCy('display-name'))
+        .invoke('val')
+        .then((val) => {
+          expect(val).equals(MOCK_PLUGIN_INFO.displayName);
+        });
+
+      // View live JSON output
+      cy.get(dataCy('open-live-view-button')).click();
+      // Check whether live JSON output now includes displayName.
+      cy.get(dataCy('live-json')).contains(`"display-name": "${MOCK_PLUGIN_INFO.displayName}",`);
+      // Check whether JSON filename is now ${pluginName}-${pluginType}.json.
+      cy.get(dataCy('plugin-json-filename')).contains(
+        `${MOCK_PLUGIN_INFO.pluginName}-${MOCK_PLUGIN_INFO.pluginType}.json`
+      );
+
+      // Button that moves to the next page is now enabled,
+      // since all the required fields are filled.
+      cy.get(dataCy('next-step-button')).should('not.be.disabled');
+
+      // Move to the next page
+      cy.get(dataCy('next-step-button')).click();
+    });
+
+    it('should edit ConfigurationGroupsPage', () => {
+      cy.get(dataCy('collapse-live-view-button')).click({ force: true });
```

Comment:
Why do we need the force:true?

Reply:
I think force: true for `collapse-live-view-btn` is needed because it's inside a material ui drawer component!